RetailCoder.VBE/Common/ExportFormatter.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Rubberduck.Common
+{
+    public static class ExportFormatter
+    {
```

Comment:
Could this be tweaked into an object that implements some `IExportFormatter` interface, constructor-injected as such wherever it's needed? The Test Explorer also uses `IClipboardWriter`...